### ISSUE TITLE
Support for Else conditions

### ISSUE THREAD

**Issue by johnkievlan** [MassTransit/Automatonymous#52]
You know what would be nice? IfElse :)

**Comment by phatboyg** [comment]
Yeah, that might be cool.

**Comment by mattsve** [comment]
Meanwhile this extension method could be of use.
```
public static EventActivityBinder<TInstance, TData> IfThenElse<TInstance, TData>(
        this EventActivityBinder<TInstance, TData> binder,
        StateMachineCondition<TInstance, TData> condition,
        Func<EventActivityBinder<TInstance, TData>, EventActivityBinder<TInstance, TData>> thenActivityCallback,
        Func<EventActivityBinder<TInstance, TData>, EventActivityBinder<TInstance, TData>> thenActivityCallback)
        where TInstance : class
    {
        var result = false;
        return binder.If(context => result = condition(context), thenActivityCallback).If(_ => !result, thenActivityCallback); 
    }
```

**Comment by phatboyg** [comment]
Fixed by #55
